src/edit_view.rs

```diff
@@ -313,8 +301,9 @@ struct FindReplace {
 impl FindReplace {
     /// Loads the glade description of the window, and builds gtk-rs objects.
     fn new() -> FindReplace {
-        let src = include_str!("ui/find_replace.glade");
-        let builder = Builder::new_from_string(src);
+        const SRC: &'static str = include_str!("ui/find_replace.glade");
```

Comment:
Well, this is very nitpicky, especially since it was called `src` before, but I guess FRGLADE_SRC (or something similar) would be clearer here.

Reply:
Well, I think considering the scope of the variable (FindReplace::new()), and the fact that the filename is referenced in the same line makes it abundantly clear - but I can change it if you insist. :)

Reply:
Alright, we can keep it like this then.